ts/features/it-wallet/screens/credentials/ItwCredentialPreviewScreen.tsx

```diff
@@ -90,7 +102,7 @@ const ItwCredentialPreviewScreen = () => {
       buttonProps: {
         label: I18n.t("global.buttons.add"),
         accessibilityLabel: I18n.t("global.buttons.add"),
-        onPress: () => null // TODO(SIW-449): Add navigation to the PIN screen
+        onPress: () => addOnPress() // TODO(SIW-449): Add navigation to the PIN screen
```

Comment:
The pin screen could be handled by a generator of the action provided?

Reply:
Yes I think so. There's no need to add another reducer and actions in my opinion though. We might want to merge them into the existing reducer. However, I have a few plans to simply the whole state structure which we can discuss later.